xo-install.sh

```diff
@@ -149,13 +149,29 @@ function InstallDependenciesDebian {
 
 	# Install necessary dependencies for XO build
 
-	if [[ $OSVERSION =~ (16|18) ]]; then
+	if [[ $OSVERSION =~ (16|18|20) ]]; then
 		echo -ne "${PROGRESS} OS Ubuntu so making sure universe repository is enabled"
 		add-apt-repository universe >/dev/null
 		echo -e "\r${OK} OS Ubuntu so making sure universe repository is enabled"
 		echo
 	fi
 
+	if [[ $OSVERSION =~ (16|18) ]]; then
+                echo -ne "${PROGRESS} Installing Python for Ubuntu 16 and 18"
```

Comment:
```suggestion
                echo -ne "${PROGRESS} Installing Python"
```
no need to specify OS version